ReCaptchaPaypal/etc/frontend/di.xml

```diff
@@ -17,7 +17,7 @@
     <type name="Magento\Checkout\Model\CompositeConfigProvider">
         <arguments>
             <argument name="configProviders" xsi:type="array">
-                <item name="recaptcha_config_provider" xsi:type="object">Magento\ReCaptchaPaypal\Model\CheckoutConfigProvider</item>
+                <item name="paypal_recaptcha_config_provider" xsi:type="object">Magento\ReCaptchaPaypal\Model\CheckoutConfigProvider</item>
```

Comment:
It looks like this should be namespaced as `magento_recaptcha_config_provider` or maybe `magento_paypal_recaptcha_config_provider` and the provider in `paypal/module-braintree-core` should be namespaced as `paypal_recaptcha_config_provider`.

Reply:
```suggestion
                <item name="magento_paypal_recaptcha_config_provider" xsi:type="object">Magento\ReCaptchaPaypal\Model\CheckoutConfigProvider</item>
```